x-pack/filebeat/module/okta/system/ingest/pipeline.yml

```diff
@@ -260,6 +264,30 @@ processors:
       value_split: "="
       target_field: _risk_object
       if: ctx.okta?.debug_context?.debug_data?.flattened?.risk != null
+      on_failure:
+        - remove:
+            field: _risk_object
```

Comment:
Not sure if the field `_risk_object` is going to exist in case `kv` processor fails here, since thats the target field 🤔
Just needed clarification as I'm not entirely sure 😅

Reply:
Answered here --> https://github.com/elastic/integrations/pull/4806#discussion_r1048225599